.vuepress/config.js

```diff
@@ -204,6 +212,15 @@ module.exports = {
                         '/community/about.md',
                     ],
                 },
+                {
+                    title: 'Guides',
+                    collapsable: false,
+                    children: [
+                        '/guides/disabling-reCAPTCHA.md',
+                        '/guides/mounts.md',
+                        '/guides/s3-backups.md'
```

Comment:
I think we could move the s3-backups stuff into an "Additional Configuration" document in the Panel section, since it's only about configuration.
We should have it be more general then though, more like

Configure Backups
there are several options ...
Use AWS S3
< what you have already >

Reply:
I agree. Just wasn't exactly sure where to place it all